### PR TITLE
feat: [PL-23207]: with account id for few API calls

### PR DESCRIPTION
**Pull request by swarajcheguri-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/20-rbac/modals/TokenModal/views/TokenForm.tsx [hunk 1]
```diff
@@ -46,7 +46,34 @@ interface TokenModalData {
 interface TokenFormData extends TokenDTO {
   expiryDate?: string
 }
-
+interface SubmitOrCloseBtn {
```
Comment (swarajcheguri-harness) [inline src/modules/20-rbac/modals/TokenModal/views/TokenForm.tsx:49]: Made this change to make sonarqube code complexity work

File: src/modules/10-common/components/Smtp/views/StepCredentials.tsx [hunk 2]
```diff
@@ -42,15 +42,25 @@ const StepCredentials: React.FC<StepProps<NgSmtpDTO> & SmtpSharedObj & CreateSmt
...
+  })
   const save = async (data: NgSmtpDTO): Promise<void> => {
     try {
       const uuid = detailsData?.uuid || prevStepData?.uuid
       const returnValue =
         uuid || isEdit
-          ? await updateSmtp({ ...data, uuid, accountId }, { headers: { 'content-type': 'application/json' } })
-          : await createSmtpConfig({ ...data, accountId }, { headers: { 'content-type': 'application/json' } })
+          ? await updateSmtp(
+              { ...data, uuid, accountId },
```
Comment (avinashmadhwani02) [inline src/modules/10-common/components/Smtp/views/StepCredentials.tsx:57]: We are already passing accountId in pageBody, why do we need it in query params also?

**Comment by abhinavrastogi-harness** [comment]
@swarajcheguri-harness please see https://harness.slack.com/archives/C02FTLMTQBA/p1647495165445759

instead of adding `accountIdentifier` to each API separately, we can add it at a framework level, and remove `routingId`

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]
@swarajcheguri-harness please see https://harness.slack.com/archives/C02FTLMTQBA/p1647495165445759

instead of adding accountIdentifier to each API separately, we can add it at a framework level, and remove routingId

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request closed without merging**